engine/src/test/scala/cromwell/engine/workflow/CallCachingBlacklistManagerSpec.scala

```diff
@@ -0,0 +1,126 @@
+package cromwell.engine.workflow
+
+import akka.event.NoLogging
+import com.typesafe.config.ConfigFactory
+import cromwell.backend.standard.callcaching.{GroupingBlacklistCache, RootWorkflowBlacklistCache}
+import cromwell.core.{HogGroup, WorkflowId, WorkflowOptions, WorkflowSourceFilesWithoutImports}
+import cromwell.engine.workflow.workflowstore.{Submitted, WorkflowToStart}
+import org.scalatest.{FlatSpec, Matchers}
+import spray.json._
+
+
+class CallCachingBlacklistManagerSpec extends FlatSpec with Matchers {
+  behavior of "CallCachingBlacklistManager"
+
+  //noinspection RedundantDefaultArgument
+  val workflowSourcesNoGrouping = WorkflowSourceFilesWithoutImports(
+    workflowSource = None,
+    workflowUrl = None,
+    workflowRoot = None,
+    workflowType = None,
+    workflowTypeVersion = None,
+    inputsJson = "",
+    workflowOptions = WorkflowOptions(JsObject.empty),
+    labelsJson = "",
+    workflowOnHold = false,
+    warnings = List.empty
+  )
+
+  val workflowSourcesYesGrouping = workflowSourcesNoGrouping.copy(
+    workflowOptions = WorkflowOptions(""" { "google_project": "blacklist_group_testing" } """.parseJson.asJsObject)
+  )
+
+  val workflowNoGrouping =
+    WorkflowToStart(
+    id = WorkflowId.randomId(),
+    submissionTime = null,
+    sources = workflowSourcesNoGrouping,
+    state = Submitted,
+    hogGroup = HogGroup("oink")
+  )
+
+  val workflowYesGrouping1 = workflowNoGrouping.copy(
+    id = WorkflowId.randomId(),
+    sources = workflowSourcesYesGrouping
+  )
+
+  val workflowYesGrouping2 = workflowYesGrouping1.copy(
+    id = WorkflowId.randomId()
+  )
+
+  it should "be off by default" in {
+    val configString = ""
+    val manager = new CallCachingBlacklistManager(ConfigFactory.parseString(configString), logger = NoLogging)
+
+    manager.blacklistCacheFor(workflowNoGrouping) shouldBe None
+  }
+
+  it should "be on with default values if blacklisting is enabled" in {
```

Comment:
I think for completeness you may also want `should "be off if blacklisting is not enabled"`

Reply:
is that different from the existing `it should "be off by default"`?